wp-h5p-xapi.php

```diff
@@ -19,10 +19,10 @@
 
 function h5pxapi_enqueue_scripts()
 {
-    wp_register_script("wp-h5p-xapi", plugins_url() . "/wp-h5p-xapi/wp-h5p-xapi.js", array("jquery"));
+    wp_register_script("wp-h5p-xapi", plugins_url() . "/wp-h5p-xapi/wp-h5p-xapi.js", array("jquery"), filemtime(), true);
```

Comment:
@SteelWagstaff `filemtime()` needs to reference a file, so (as per the example in your issue) it should be something like:

```diff
- wp_register_script("wp-h5p-xapi", plugins_url() . "/wp-h5p-xapi/wp-h5p-xapi.js", array("jquery"), filemtime(), true);
+ wp_register_script("wp-h5p-xapi", plugins_url() . "/wp-h5p-xapi/wp-h5p-xapi.js", array("jquery"), filemtime( __FILE__ ), true);